torch/utils/data/datapipes/utils/common.py

```diff
@@ -78,6 +77,11 @@ def validate_input_col(fn: Callable, input_col: Optional[Union[int, tuple, list]
         else:
             continue
 
+    if isinstance(fn, functools.partial):
+        fn_name = fn.func.__name__
+    else:
+        fn_name = fn.__name__
```

Comment:
This causes lots of problem as not all of callable objects have `__name__`
I am going to do a forward fix.

Reply:
Simple example would be a `nn.Module`